### DISCUSSION TITLE
Error in  get_pin_file options

### DISCUSSION THREAD

**Discussion by adolfoisla** [egeulgen/pathfindR#83] [Q&A] [answered]
Dear, I'm using the tool to analyze non-human samples (Danio rerio), but when I use 
"## retrieving PIN data 
path_to_pin_file <- get_pin_file(source = "BioGRID", org = "Danio_rerio")

but the result is :
Error in utils::download.file(biogrid_url, tmp, method = "wget", quiet = TRUE) : 
  'wget' call had nonzero exit status.

how can I fix this error?

Regards

**Comment by egeulgen** [comment, marked as answer]
hey @adolfoisla,

the default method for downloading the BioGRID PIN file was using "wget", I've updated that to be "auto" (selects a downloader based on your system) but this might actually be a proxy-related issue, do you  use a proxy to connect to the internet?  If so, this page might help: https://support.rstudio.com/hc/en-us/articles/200488488-Configuring-R-to-Use-an-HTTP-or-HTTPS-Proxy

Please download the latest development version of pathfindR and try to see whether the fix I implemented works for you:
```r
install.packages("devtools") # if you have not installed "devtools" 
devtools::install_github("egeulgen/pathfindR")
```
